src/components/entity/ha-state-label-badge.ts

```diff
@@ -162,7 +164,9 @@ export class HaStateLabelBadge extends LitElement {
           ? "hass:timer-outline"
           : "hass:timer-off-outline";
       default:
-        return null;
+        return state?.attributes.device_class === "moon__phase"
```

Comment:
Why do we have this icon logic in here, instead of using the global icon functions? 🤔